### DISCUSSION TITLE
How can I reference nvconfig settings in my config files?

### DISCUSSION THREAD

**Discussion by IcyAlmond** [NvChad/NvChad#3058] [Q&A]
I want to change the statusline a little bit. I want to edit this cursor line https://github.com/NvChad/ui/blob/3839b74f58602bec36db8150333c8b09966e42ed/lua/nvchad/stl/default.lua#L54 and have it persist between updates.
I've added this to my configuration:
```lua
local config = require("nvconfig").ui.statusline
local sep_style = config.separator_style
local utils = require "nvchad.stl.utils"
local sep_icons = utils.separators
local separators = (type(sep_style) == "table" and sep_style) or sep_icons[sep_style]
local sep_l = separators["left"]

M.ui = {
  statusline = {
    modules = {
      cursor = "%#St_pos_sep#" .. sep_l .. "%#St_pos_icon# %#St_pos_text# %p %% "
    }
  },
}
```
But it seems that using the require("nvconfig") at the top make it so that my custom theme is overridden back to default. So I tried this instead replacing config.separator_style with what ui.statusline.separator_config is manually https://github.com/NvChad/ui/blob/3839b74f58602bec36db8150333c8b09966e42ed/lua/nvconfig.lua#L31 :
```lua
local sep_style = "default"
local utils = require "nvchad.stl.utils"
local sep_icons = utils.separators
local separators = (type(sep_style) == "table" and sep_style) or sep_icons[sep_style]
local sep_l = separators["left"]

M.ui = {
  statusline = {
    enabled = true,
    modules = {
      cursor = "%#St_pos_sep#" .. sep_l .. "%#St_pos_icon# %#St_pos_text# %p %% "
    }
  },
}
```
This works, but is there a way to not have to hardcode the word "default" here? and do it the same way as the original cursor file did it?

**Comment by siduck** [comment]
you are not supposed to import nvconfig. Please read :h nvui
